{{ cookiecutter.project_slug }}/backend/Makefile

```diff
@@ -71,16 +71,17 @@ clean-test: ## remove test and coverage artifacts
 bin/pip:
 	@echo "$(GREEN)==> Setup Virtual Env$(RESET)"
 	python3 -m venv .
-	bin/pip install -U "pip" "wheel" "cookiecutter"
+	bin/pip install -U "pip" "wheel" "cookiecutter" "mxdev"
 
 instance/etc/zope.ini:	bin/pip
 	@echo "$(GREEN)==> Install Plone and create instance$(RESET)"
-	bin/pip install Plone -c https://dist.plone.org/release/$(PLONE_VERSION)/constraints.txt
+	bin/mxdev -c mx.ini
+	bin/pip install -r requirements-mxdev.txt
```

Comment:
I suggest to move these two lines to `build-dev` below, replacing `bin/pip install -r requirements/dev.txt`
Then a `make install` in root directory (`make start-backend`) succeeds.

Reply:
both `make build-dev` and `make build`.
https://github.com/collective/cookiecutter-plone-starter/pull/7#issuecomment-1253343089
don't know why both are needed, but this PR is a minimal invasive change before mxmkae, is'nt it?